modules/ROOT/pages/bind-api-configure-service-mesh.adoc

```diff
@@ -4,11 +4,9 @@ include::_attributes.adoc[]
 endif::[]
 :keywords: service mesh, microservices, binding
 
-When you create the binding, Anypoint Service Mesh creates a three-way association of the service in Kubernetes with the API that you created in Anypoint Platform to represent the service, and the adapter that you created for the service. The adapter uses the unique API ID of the API to fetch policies and apply them to your services that are running in the cluster.
+When you create the binding, Anypoint Service Mesh creates a three-way association of the service in Kubernetes, the API that you created in Anypoint Platform to represent the service, and the adapter that you created for the service. The adapter uses the unique API ID to fetch policies and apply them to your services that are running in the cluster.
 
-After you create the binding, Anypoint Service Mesh takes few seconds to complete the binding procedure and updates the status.
-
-After the binding is completed, you can view the status using the `list` command. The status values include:
+When you create the binding, Anypoint Service Mesh takes few seconds to complete the binding procedure and updates the status. After the binding is completed, you can view the status using the `list` command. The status values include:
```

Comment:
update (parallel with "complete" before it. 
Subject: ASM
Predicate: takes
--- Infinitive: to complete the binding and update the status

Reply:
Yes, you are right! Good catch!